packages/ui/src/components/style/core/global.less

```diff
@@ -6,4 +6,16 @@
 :root {
   --condo-font-fallback: sans-serif;
   --condo-font-fallback-mono: consolas, monaco, "Andale Mono", "Ubuntu Mono", monospace;
-}
\ No newline at end of file
+}
+
+@from-mobile-small: ~"screen and (min-width: 0px)";
```

Comment:
Does it mean that syntax is:
```
@media @from-mobile-small and @to-tablet-large
```
and not like?
```
@media @from-mobile-small @to-tablet-large
@media @from-mobile-small
@media @mobile-small
```
as proposed originally?
what was the concerns?

Reply:
If we do the second option, we will not be able to write a media query like this: `@media @to-tablet-large` because `@to-tablet-large` starts with `and`.